roxie/ccd/ccdqueue.cpp

```diff
@@ -2790,7 +2790,7 @@ class RoxieSocketQueueManager : public RoxieReceiverBase
                 {
 #ifdef NEW_IBYTI
                     unsigned delay = 0;
-                    if (mySubchannel != 0)  // i.e. I am not the primary here
+                    if (mySubchannel != 0 && (header.activityId & ~ROXIE_PRIORITY_MASK) < ROXIE_ACTIVITY_SPECIAL_FIRST)  // i.e. I am not the primary here, and never delay special
```

Comment:
observation:  Adding isSpecialActivity(headerActivityId) would make code like this easier to read.